tests-e2e/cypress/integration/playbooks/creation_button_spec.js

```diff
@@ -61,6 +61,11 @@ describe('playbooks > creation button', () => {
 
         // * Verify playbook outline page opened
         verifyPlaybookOutlineOpened(playbookName);
+
+        // * Verify playbook was added to the LHS
+        cy.findByTestId('lhs-navigation').within(() => {
+            cy.findByText(playbookName).should('exist');
+        });
```

Comment:
optional: would make sense to make this a one-liner?
```suggestion
        cy.findByTestId('lhs-navigation').findByText(playbookName).should('exist');
```